packages/fast-css-editor-react/src/position/index.tsx

```diff
@@ -0,0 +1,173 @@
+import * as React from "react";
+import { IManagedClasses } from "@microsoft/fast-components-class-name-contracts-base";
+import manageJss, { IJSSManagerProps } from "@microsoft/fast-jss-manager-react";
+import style, { ICSSPositionClassNameContract } from "./position.style";
+
+export enum PositionValue {
+    static = "static",
+    absolute = "absolute"
+}
+
+export enum Location {
+    top = "top",
+    left = "left",
+    right = "right",
+    bottom = "bottom"
+}
+
+export interface ILocationsMappedToClassNames {
+    location: Location;
+    className: string;
+}
+
+export interface ICSSPositionProps {
+    position?: PositionValue;
+    left?: number;
+    right?: number;
+    top?: number;
+    bottom?: number;
+    onChange: (positionValues: any) => void;
+}
+
+class CSSPosition extends React.Component<ICSSPositionProps & IManagedClasses<ICSSPositionClassNameContract>, {}> {
+    private positionKey: string = "position";
+
+    public render(): JSX.Element {
+        return (
+            <div className={this.props.managedClasses.cssPosition}>
+                <span className={this.props.managedClasses.cssPosition_selectContainer}>
+                    <select
+                        className={this.props.managedClasses.cssPosition_selectContainer_select}
+                        data-location={this.positionKey}
+                        onChange={this.handleOnChange}
+                        value={this.props.position ? this.props.position : PositionValue.static}
+                    >
```

Comment:
why not relative and fixed?

Reply:
Per @Falkicon we are only doing static and absolute at this time.